ur_robot_driver/include/ur_robot_driver/ros/hardware_interface.h

```diff
@@ -257,6 +257,8 @@ class HardwareInterface : public hardware_interface::RobotHW
   bool controllers_initialized_;
 
   std::string robot_ip_;
+  int reverse_port_;
+  int script_sender_port_;
```

Comment:
Do we need to keep them as members? I think, we only pass them into the `URDriver` constructor, right?

Reply:
Right, removed.